Test/Test.csproj

```diff
@@ -2,11 +2,13 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>

```

Comment:
Do we need unsafe blocks in the tests?

Reply:
Yes. I had to create a Mockup class to check that Tensor<T>[i] is pointing to the same object as TorchTensor[i].